docs/variants/novacustom_ns5x_adl/firmware-update.md

```diff
@@ -0,0 +1,37 @@
+# Firmware update
+
+The following documentation describes the process of Dasharo open-source
+firmware update. The update process may be different, depending on which
+firmware version is currently installed on your device. The currently installed
+firmware version can be checked with the following command in a Linux
+environment:
+
+```shell
+sudo dmidecode -t bios | grep Version
+```
+
+Alternatively, it can be checked in the `BIOS Setup Menu`.
+
+## Version v1.0.0
+
+You should update to the [v1.0.0 version](../releases/TBD) first by flashing
+externally to overwrite ME region and install Dasharo, then update EC by
+following [this procedure](initial-deployment.md).
+
+ME region is provisioned for Boot Guard which must be disabled first. Also
+software flashing is not possible due to enabled BIOS Guard.
```

Comment:
This part may be generic answer at some point to question "Why I have to use external SPI programmer to deploy Dasharo on my latop?" or something like that.
It could be more elaborated.

Reply:
This section has been changed to:

```
    During the initial installation of Dasharo, you should deploy the supported
    Intel ME version (and configuration) on the device. Since vendor firmware
    has enabled Intel Boot Guard and BIOS Guard, it is not possible to do this
    from within the operating system and external flashing of the whole flash
    chip using a programmer like the CH341a is required.
```

in the combined initial deployment doc.